### PR TITLE
OKTA-440888: Custom pages macros update

### PR DESCRIPTION
**Pull request by arsalanbadar-okta**
## Description:
- **What's changed?**
Adding macros related to THEME_BUILDER
- **Is this PR related to a Monolith release?** 
2021.11

Monolith changes: 
https://github.com/okta/okta-core/pull/58584

### Resolves:

* [OKTA-440888](https://oktainc.atlassian.net/browse/OKTA-440888)

### CODE REVIEW

**Review by albertchen-okta — APPROVED** [review]

**Review by brentschaus-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Example:
 
 ### Add your own scripts
 
-You can add custom JavaScript code to the sign-in page. In the embedded HTML editor, add a script in the `<head>` section, below the <span v-pre>`{{{SignInWidgetResources}}}`</span> line.
+You can add custom JavaScript code to the sign-in page. In the embedded HTML editor, add a script in the `<head>` section, below the `SignInWidgetResources` line.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md:29]:
```suggestion
You can add custom JavaScript code to the sign-in page. In the embedded HTML editor, add a script in the `<head>` section, below the `{{{SignInWidgetResources}}}` line.
```

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Use the following examples to help you customize the sign-in page with your own
 
 ### Add your own stylesheet
 
-You can add your own stylesheet to extend the look of the sign-in page. In the embedded HTML editor, add a link to your stylesheet in the `<head>` section, below the <span v-pre>`{{{SignInWidgetResources}}}`</span> line.
+You can add your own stylesheet to extend the look of the sign-in page. In the embedded HTML editor, add a link to your stylesheet in the `<head>` section, below the `SignInWidgetResources` line.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md:9]:
```suggestion
You can add your own stylesheet to extend the look of the sign-in page. In the embedded HTML editor, add a link to your stylesheet in the `<head>` section, below the `{{{SignInWidgetResources}}}` line.
```

**Review by brentschaus-okta — APPROVED** [review]

**Comment by brianduffield-okta** [comment]
Re-opening PR to do some internal testing.

**Comment by brianduffield-okta** [comment]
TEST: Removing double curly braces and span tags to see if this fixes Travis build failure. 

Will re-close this PR afterwards. The content of this PR was ported to: https://github.com/okta/okta-developer-docs/pull/2718

**Comment by brianduffield-okta** [comment]
TEST Successful!! Travis builds cleanly. Closing this PR.

**Pull request closed without merging**